Vagrantfile

```diff
@@ -71,15 +72,17 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       --foreman-db-type='postgresql' --foreman-db-database='foreman' --foreman-db-host='pg1.localdomain' \
       --foreman-db-manage=false --foreman-db-username='foremandbuser' --foreman-db-password='Y3ll0-h@t' \
       --puppet-dns-alt-names='puppet.localdomain' \
-      --foreman-passenger-interface='172.28.128.20'
+      --foreman-passenger-interface='172.28.128.20' ; true
     EOF
 
     foreman.vm.provision "shell", inline: "echo '*.localdomain' > /etc/puppetlabs/puppet/autosign.conf"
 
-    foreman.vm.provision "shell", inline: "puppet module install theforeman-puppet"
+    #foreman.vm.provision "shell", inline: "puppet module install theforeman-puppet"
+    foreman.vm.provision "shell", inline: "yum install -y git; git clone https://github.com/theforeman/puppet-puppet.git /etc/puppetlabs/code/environments/production/modules/puppet"
     foreman.vm.provision "shell", inline: "puppet module install theforeman-foreman"
     foreman.vm.provision "shell", inline: "puppet module install puppetlabs-puppetdb"
     foreman.vm.provision "shell", inline: "puppet module install puppet-r10k"
+    foreman.vm.provision "shell", inline: "sed -i  '13i\"puppetlabs.trapperkeeper.services.metrics.metrics-service/metrics-webservice\": \"/metrics\"' /etc/puppetlabs/puppetserver/conf.d/web-routes.conf"
```

Comment:
Yup, but it looks like it should be fixed in the next release according to theforeman/puppet-puppet#509